pkg/types/v1/image.go

```diff
@@ -29,7 +29,7 @@ type OCIImageExtractor struct{}
 
 var _ ImageExtractor = OCIImageExtractor{}
 
-func (e OCIImageExtractor) ExtractImage(imageRef, destination, platformRef string) error {
+func (e OCIImageExtractor) ExtractImage(imageRef, destination, _ string) error {
 	img, err := utils.GetImage(imageRef, utils.GetCurrentPlatform(), nil, nil)
```

Comment:
shouldn't we be doing some check that the platformRef matches the current platform or something like that? You already did something like that in the kairos-sdk: https://github.com/kairos-io/kairos-sdk/pull/557/files#diff-5a53c19741f79228c7414fd4f325e7d79935eb2b5c051ae2e44c0d9521542006R111

or is that not the same case?

Reply:
IIRC we are using the sdk here as a backend, this methods were here becuase it was part of the agent before we put in on the sdk and then we moved but could not update or something so we leaved this stub in here. we should be using the sdk ones directly!